Jenkinsfile

```diff
@@ -1140,7 +1140,7 @@ pipeline {
                                   rm -rf "Functional/"
                                   mkdir "Functional/"
                                   # compress those potentially huge DAOS logs
-                                  if daos_logs=$(ls install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/*); then
+                                  if daos_logs=$(find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size +1M); then
```

Comment:
While I agree that it's no worse than it was, I didn't realize how bad it was, but I wonder far away from an _Argument list too long_.  :open_mouth:   Here's an example from your test run:

```
[2020-04-03T12:43:14.685Z] ++ find install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_agent.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm6 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm6 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_agent.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-1634c07-pool-destroy_rebuild/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_agent.log-wolf-102vm8 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.34-087626f-pool-rebuild_tests/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.34-51bc187-pool-rebuild_no_cap/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-4c8a5cd-pool-rebuild_with_io/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-505a5a9-pool-rebuild_with_ior/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos_server.log-wolf-102vm2 -maxdepth 0 -type f -size +1M
```

Not something that necessarily needs fixing here, but something for us to just keep in mind and now particularly that we are using `find` options to fix it are much more plentiful.